### ISSUE TITLE
[Feature] Add note to favorite animation

### ISSUE THREAD

**Issue by Bartek532** [taniarascia/takenote#511]
**Problem**
When user add note to favorites, only star is added to note, personally, first time I missed it, so I think user should get a clearer signal

**Solution**
Make some kind of animation with star to let user know, that their offer has been added to your favorites

**Comment by Parteeksachdeva** [comment]
How do you want it? I tested it it's showing a star after bookmarking a note. Do you want to change it to some kind of animation?

**Comment by Bartek532** [comment]
Yep, I think the animation will be the best, but I am not entirely sure what it should look like 😛

**Comment by Parteeksachdeva** [comment]
ok, sure I will work on it.

**Comment by sayna3311** [comment]
Will it be better, if the star turned in came to its place?

**Comment by Bartek532** [comment]
Hmm, maybe 😛

**Comment by sayna3311** [comment]
Should I implement this and send a PR?

**Comment by Bartek532** [comment]
Yeah 👍
